bin/cron/process_foorm_data

```diff
@@ -75,7 +75,7 @@ def write_submissions_to_redshift(client, arn)
       submission_id    int,
       item_name        varchar,
       matrix_item_name varchar,
-      response_value   varchar,
+      response_value   varchar(2000),
```

Comment:
This is the simple fix. If we do just this, we could keep using the `process_foorm_data` script and skip the rest of this PR.

Reply:
is there any more benefit in making it `varchar(max)`? do we know any constraints on the response_value field?

Reply:
I think `max` is overkill at ~65536 and while `max` vs `n` should be the same from a storage perspective, for performance `max` is slower. [AWS advises against defaulting to the max column size](https://docs.aws.amazon.com/redshift/latest/dg/c_best-practices-smallest-column-size.html).

2000 is a little over 3x bigger than the size of the data that caused the problem. As far as I can tell, the maximum _possible_ size is ~65536, similar to `max`, but that's because we defaulted to the largest possible size in our mysql migration too.